climakitae/uncertain_utils.py

```diff
@@ -568,17 +657,214 @@ def ks_stat_2sample(sample1, sample2):
 
     d_statistic, p_value = xr.apply_ufunc(
         ks_stat_2sample,
-        sample1, sample2,
-        input_core_dims=[["index"],["index"]],
+        sample1,
+        sample2,
+        input_core_dims=[["index"], ["index"]],
         exclude_dims=set(("index",)),
         output_core_dims=[[], []],
     )
-        
+
+    p_df = p_value.dropna(dim="allpoints")
     p_df = p_value.rename("p_value")
     p_df = p_df.unstack("allpoints")
     p_df = p_df.to_dataframe().reset_index()
-    p_df = p_df[["lat","lon","p_value"]]
-    p_df = p_df.loc[:,['lon','lat','p_value']]
+    p_df = p_df[["lat", "lon", "p_value"]]
+    p_df = p_df.loc[:, ["lon", "lat", "p_value"]]
     p_df = p_df[p_df["p_value"] < sig_lvl]
-    
+
     return p_df
+
+
+def get_warm_level(warm_level, ds, multi_ens=False, ipcc=True):
```

Comment:
Getting the following error in both uncertainty notebooks when trying to run get_warm_level _by itself:_ 
`'int' object is not iterable
`

Reply:
Additionally, at least in the model uncertainty notebook, when passing a warm_level and a ds (with time, x, y, simulation dims) it returns nothing and doesn't print anything out, and this does not seem to be the desired behavior. Testing over in the precip notebook, I can test what it outputs, but I'm not sure in the compatibility to the other uncertainty notebook.

Reply:
thanks for this. I'll check it out on Monday!

Reply:
If I remember correctly, the function actually requires a float input, not an int-- i.e. you need to input `warm_level = 4.0` (a float) vs. `warm_level = 4` (an int)

Reply:
Should we handle `int` as well as `float` inputs?